docs/_shared/Ad.tsx

```diff
@@ -25,14 +49,17 @@ const useStyles = makeStyles(theme => ({
 
 const Ad: React.FC = () => {
   useStyles();
+
   React.useEffect(() => {
-    const codefundScriptPosition = document.querySelector('#codefund-script-position');
+    const adScriptPosition = document.querySelector('#ad-script-position');
 
-    if (codefundScriptPosition) {
-      loadScript(
-        'https://codefund.io/properties/197/funder.js?theme=light',
-        codefundScriptPosition
+    if (adScriptPosition) {
+      let script = loadScript(
+        '//cdn.carbonads.com/carbon.js?serve=CE7ILK7U&placement=material-ui-pickersdev',
```

Comment:
```suggestion
        'https://cdn.carbonads.com/carbon.js?serve=CKYIL27L&placement=material-uicom',
```

Reply:
No it will use this link before merging docs